index.html

```diff
@@ -1,40 +1,180 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mobile version Replica</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./style.css">
-
-
-</head>
-<body>
-    <div class="container">
-        <header>
-            <nav class="navbar">
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-            </nav>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+        />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="style.css" />
+        <title>Portfolio</title>
+    </head>
+    <body>
+        <header class="bkg">
+            <button class="menu">
+                <span class="material-symbols-outlined"> menu </span>
+            </button>
         </header>
+        <!-- Headline -->
+        <section class="spacing headline">
```

Comment:
Hello again @Kamba56 ! Maybe you can help me because in my browser I can see all the images and I don't understand why you don't
![Screenshot (2)](https://user-images.githubusercontent.com/116699473/206881955-035928f1-a7ae-4eca-bd1d-b898b3c51af7.png)